explorer-api/app/cardano-explorer-api.hs

```diff
@@ -1,5 +1,49 @@
+{-# LANGUAGE OverloadedStrings #-}
 
-import Explorer.Web (runServer)
+import Cardano.Db
+    ( readPGPassFileEnv )
+import Explorer.Web
+    ( WebserverConfig (WebserverConfig), runServer )
+import Network.Wai.Handler.Warp
+    ( HostPreference, Port )
+import Options.Applicative
+    ( Parser, ParserPrefs )
+import Options.Applicative
+    ( auto, help, long, metavar, option, showDefault, value )
+import qualified Options.Applicative as Opt
 
 main :: IO ()
-main = runServer
+main = do
+  webserverConfig <- Opt.customExecParser prefs opts
+  pgConfig <- readPGPassFileEnv
+  runServer webserverConfig pgConfig
+  where
+    opts =
+      Opt.info
+        (Opt.helper <*> pConfig)
+        (Opt.fullDesc <>
+         Opt.header
+           "cardano-explorer-api - Run a reporting API server that explores the blockchain")
```

Comment:
Best to align with https://github.com/input-output-hk/cardano-rest/blob/master/explorer-api/cardano-explorer-api.cabal#L7

```suggestion
           "cardano-explorer-api - A block explorer for the cardano network")
```